extensions/wikia/AdEngine/js/video/player/jwplayer/jwplayerTracker.js

```diff
@@ -31,14 +35,34 @@ define('ext.wikia.adEngine.video.player.jwplayer.jwplayerTracker', [
 			return;
 		}
 
+		params.withCtp = !player.getConfig().autostart;
 		params.withAudio = !player.getConfig().mute;
 
 		Object.keys(trackingEventsMap).forEach(function (playerEvent) {
 			player.on(playerEvent, function(event) {
-				var errorCode;
+				var errorCode,
+					vastParams = event.tag ? vastParser.parse(event.tag) : null;
+
+				if (vastParams && vastParams.customParams) {
+					if (
+						vastParams.customParams.ctp !== undefined
+						&& params.withCtp !== (vastParams.customParams.ctp === 'yes')
+					) {
+						params.withCtp = vastParams.customParams.ctp === 'yes';
+					}
+
+					if (
+						vastParams.customParams.audio !== undefined
+						&& params.withAudio !== (vastParams.customParams.audio === 'yes')
```

Comment:
What do you think of extracting this condition to a method with a descriptive name? It may save some time in the feature when we'll get back to this code ;) What the name would be? `customParamDefinedAndFlagNotSet()`? :)